### ISSUE TITLE
Different gradient penalty from the paper

### ISSUE THREAD

**Issue by kudo1026** [wiseodd/generative-models#31]
https://github.com/wiseodd/generative-models/blob/566ac2e95971ab55d65169e565b9d53023ccc113/GAN/improved_wasserstein_gan/wgan_gp_tensorflow.py#L89

Hi,
Thanks for the code, but I found that the gradient penalty term is different from the paper and the code given by the author
`gradient_penalty = tf.reduce_mean((slopes-1.)**2)`
https://github.com/igul222/improved_wgan_training/blob/fa66c574a54c4916d27c55441d33753dcc78f6bc/gan_mnist.py#L147

However, your code still works well and produce good results. I don't know if you intentionally did it, or it's just a typo but fortunately make no mistakes?

Thanks

**Comment by wiseodd** [comment]
Hi, thanks for noticing it.
It was a typo. You are more than welcome to submit a pull request for this.
